docs/setup.md

```diff
@@ -5,12 +5,8 @@ SDO client-sdk uses safestringlib for string and memory operations to prevent se
 
 1. For Linux*/ Arm Mbed* OS/ Arm Mbed Linux OS builds, download safestring from <a href="https://github.com/intel/safestringlib">intel-safestringlib</a>.

```

Comment:
Formatting issue here.

Reply:
ignored as of now, was not part of this patch, pls push with the change.@prateekcr